yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala

```diff
@@ -351,11 +352,13 @@ private object YarnClusterDriver extends Logging with Matchers {
       assert(driverLogs.size === 2)
       assert(driverLogs.containsKey("stderr"))
       assert(driverLogs.containsKey("stdout"))
-      val stderr = driverLogs("stderr") // YARN puts everything in stderr.
-      val lines = Source.fromURL(stderr).getLines()
```

Comment:
Yep, I will clean the import up. Also, I think I am going to remove the check for hostname (just rely on the URL format).